docs/start/install.rst

```diff
@@ -66,44 +70,76 @@ The following platforms are supported:
 
   Windows wheels are available for Windows 10 or later.
 
-
+.. _Install Dev Version:
 
 Install latest dev version
 --------------------------
 
 If you wish to test the latest version of ``datatable`` before it has been
 officially released, then you can use one of the binary wheels that we build
-as part of our Continuous Integration process.
+as part of our Continuous Integration process. 
 
-If you are on Windows, then pre-built wheels are available on `AppVeyor`_.
-Click on a green main build of your choice, then navigate to the "Artifacts"
-tab, copy the wheel URL that corresponds to your Python version, and finally
-install it as:
+There are two sources available:
 
-.. code-block:: doscon
+- For macOS and Linux please use our `S3 repository`_. 
+
+- For Windows installations, you can find the latest wheels within our
+  `AppVeyor`_ build system.
```

Comment:
I would not call this "sources", but rather structure this section in the same way as above. Something like this may work:
```
- **macOS / Linux x86_64 / ppc64le**
  The wheels could be Installed from :ref:`datatable S3 repository<Install from datatable S3 repository>`
- **Windows**
  The wheels could be Installed from :ref:`datatable AppVeyor account<Install from datatable AppVeyor artifacts>`
```